### ISSUE TITLE
Accessibility updates for non-modal banner

### ISSUE THREAD

**Issue by edthedev** [app-illinois/Design-Resources#161]
## Context

Per today's discussion with our accessibility experts, if the banner cannot be modal, we need to make further accessibility updates to avoid a disorienting experience.

Note that the reason we have been unwilling to make the banner modal, is #140. If we can address #140 directly, then we can make the banner modal, and some of this is not needed. Some of it is still a good idea. We can reach out to KH again for additional guidance.

## Tasks

- [x] let's start by revisiting #140 - if we can solve that in a way that reliably works across our test sites, we can make it modal again.

Otherwise:

- [x] on tab out - put focus back at the top of the page - either forward or backward (call the dismiss function, and have the dismiss function put the focus back at the top of the page) 
- [x] update so that the escape key also makes the dialog go away
- ~~for now - dismiss the banner entirely when they tab out (until it can be made modal)~~ No clean way to implement this. But Escape helps.

**Comment by edthedev** [comment]
Blocked - may not be needed  - pending if https://github.com/app-illinois/Design-Resources/pull/167 is merged.

**Comment by edthedev** [comment]
Since #140 is working, we do not plan to make these changes. Rather, we will proceed to #168

**Comment by edthedev** [comment]
I feel that the chance that the zombie behavior persists for some sites is too high to make the banner modal, today. But after the switch-over, we will have insight into whether the zombie behavior is solved for all campus websites, at which time we can revisit #168.

In the meantime, I am re-opening this to revisit.
